src/index.js

```diff
@@ -242,8 +206,32 @@ class SoshContentScript extends ContentScript {
 
   async ensureNotAuthenticated() {
     this.log('info', '🤖 ensureNotAuthenticated starts')
-    await this.goto(LOGOUT_URL)
-    await this.waitForElementInWorker('#oecs__connexion')
+    await this.goto(DEFAULT_PAGE_URL)
+    await this.waitForElementInWorker(
+      '#login, #password, div[class*="captcha_responseContainer"], button[data-testid="button-keepconnected"], #oecs__connexion, #oecs__connecte-se-deconnecter'
+    )
+    await this.handleCaptcha()
```

Comment:
This is weird to have the handle of captcha in ensureNotAuthenticated since we did not try any authentication yet.

Reply:
I know but it happens during the development, so at least we're covering the case 🤷

Reply:
Ok, let's add a comment to know the reason for this then